pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -120,6 +160,30 @@ def __init__(
 
 
 class ConvSequence(SequentialWithOutChannels):
+    r"""Sequence of convolutional blocks that occurs repeatedly in :cite:`ULVL2016`.
+
+    Each sequence contains three ``ConvBlock`` from :cite:`ULVL2016`:
+
+    * A convolution with ``in_channels`` as input and ``out_channels`` as output and a
+      ``kernel_size=3``.
+    * A convolution with ``out_channels`` as input and output and a ``kernel_size=3``.
+    * A convolution with ``out_channels`` as input and output and a ``kernel_size=1``.
```

Comment:
I think you can shorten this to say "three sequential conv blocks while the first two have a kernel size of 3 ...". Conv block should link to the class defined above.